### DISCUSSION TITLE
Integration with Langchain, Weaviate, LlamaIndex

### DISCUSSION THREAD

**Discussion by 99bits** [Ironclad/rivet#191] [Q&A]
Hi Friends, 

I recently found Rivet and it's a game changer tool.  I watched all of the videos and went through the docs.  Very impressed.  Currently our project is using Langchain, LlamaIndex and Weaviate.  Would love to get some advice of how people are integrating Rivet with either Langchain, LlamaIndex or Weaviate.  

Cheers.

**Comment by abrenneke** [comment]
I'd also ask in the discord @99bits!
